frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -222,7 +244,22 @@ export default React.memo(function MessageEditor({
           <div>{i18n.messages.messageEditor.sender}</div>
         </div>
         <Select
-          options={senderOptions}
+          options={accounts
+            .filter((account: MessageAccount) => {
+              if (account.type === 'PERSONAL') {
+                return true
+              } else if (account.type === 'GROUP') {
+                return !!getSelectorName(
+                  account.daycareGroup.id,
+                  availableReceivers
+                )
+              }
+              return false
+            })
```

Comment:
And on the contrary, when returning a value like here, I would prefer an expression like 
```typescript
(account: MessageAccount) =>
  account.type === 'PERSONAL' ||
  (account.type === 'GROUP' && !!getSelectorName(account.daycareGroup.id, availableReceivers))
```

Reply:
I would like to have this logic to be extracted outside of JSX.  `const senderOptions = accounts.filter(..)`